lib/resources/member-saved-items.js

```diff
@@ -0,0 +1,204 @@
+ *
+ * @param {number} programId The ID of the item's parent program
+ *
+ * @returns {module:resources/items~GetAllFunc} Returns `getAll` program items function.
+ *
+ * @see {@link module:resources/items~GetAllFunc} for examples.
+ *
+ * @protected
+ */
+const getAllProgramItems = programId => {
+  return function getAll(params, callback) {
+    const options = {
+      path: `/programs/${programId}/member_saved_items`,
+      config: config.getAll(),
+    };
+
+    /*
+     * If called with two arguments,
+     * set options.params to the first
+     * argument (`params`).
+     *
+     * Otherwise (i.e., called with just
+     * one argument), set the callback as
+     * the argument.
+     */
+    if (arguments.length === 2) {
+      options.params = params;
+    } else {
+      // eslint-disable-next-line no-param-reassign
+      callback = params;
+    }
+
+    api.get(options, callback);
+  };
+};
+
+/**
+ * Function for getting an array of item JSON objects.
+ *
+ * @typedef module:resources/items~GetAllFunc
+ *
+ * @property {object} params [Request]{@link https://github.com/request/request} params to pass to the API call
+ * @property {module:api~requestCallback} callback Callback that handles the response
+ *
+ * @example
+ * // Gets an array of item detail objects the program with id 12
+ * ro.program(12).items.getAll((error, responseBody, response) => {
+ *   // ...
+ * });
+ */
+
+/**
+ * Higher order function for creating a `get` program item function.
+ *
+ * @param {number} programId The ID of the item's parent program
+ *
+ * @returns {module:resources/items~GetFunc} Returns `get` program item function.
+ *
+ * @see {@link module:resources/items~GetFunc} for examples.
+ *
+ * @protected
+ */
+const getProgramItem = programId => {
+  return function get(itemId, params, callback) {
+    const options = {
+      path: `/programs/${programId}/member_saved_items/${itemId}`,
+      config: config.getAll(),
+    };
+
+    /*
+     * If called with three arguments,
+     * set options.params to the second
+     * argument (`params`).
+     *
+     * Otherwise (i.e., called with just
+     * two arguments), set the callback as
+     * the second argument.
+     */
+    if (arguments.length === 3) {
+      options.params = params;
+    } else {
+      // eslint-disable-next-line no-param-reassign
+      callback = params;
+    }
+
+    api.get(options, callback);
+  };
+};
+
+/**
+ * Higher order function for creating a `getParameters` program item function.
+ *
+ * @param {number} programId The ID of the item's parent program
+ *
+ * @returns {module:resources/items~GetParametersFunc} Returns `getParameters` program item function.
+ *
+ * @see {@link module:resources/items~GetParametersFunc} for examples.
+ *
+ * @protected
+ */
+const getProgramItemParameters = programId => {
+  return function getParameters(params, callback) {
+    const options = {
+      path: `/programs/${programId}/member_saved_items/parameters`,
```

Comment:
Insofar as I'm aware, this API endpoint does not exist in Pangea. @benevolentmadman can you please confirm?

I haven't reviewed the PR for PAN-9228 thoroughly but I don't see it in https://github.com/rewardops/pangea/pull/8805/files#diff-9a2d7742cc8fe7c5ba549fe64b6502a7ffa9dbf97b310e7fbab7b8c0064947ba

Reply:
I just copy/pasted get items file in SDK and changed the `items` to `member_saved_items`, but if this does not exist, then I'll delete these.

Reply:
this one does not exist

Reply:
Okay, I'll remove it from here.